images/generate-assets.sh

```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-# These assets are used almost exclusively for IE8 fallback, so don't need retina versions
```

Comment:
Is there another change to this file I'm missing to generate the @2x and @3x versions?